src/Task/index.js

```diff
@@ -12,20 +12,21 @@ const TaskIcon = ({ variant }) => {
   }
 }
 
-const TaskTitle = ({ title, htmlTitle }) => {
-  return  htmlTitle ? (
-    <div dangerouslySetInnerHTML={{__html: title}} />
-  ) : (
-    <div>{title}</div>
-  )
-}
+const TaskTitle = ({ title, htmlTitle, extra }) => 
+  <>
+    {htmlTitle ?
+      <span className="ola_task-title-text" dangerouslySetInnerHTML={{__html: title}} /> :
+      <span className="ola_task-title-text">{title}</span>
+    }
+    {extra && <div className="ola_task-title-extra">{extra}</div>}
+  </>
```

Comment:
I'd rename `ola_task-title-text` and `ola_task-title-extra` to simply `ola_task-title` and `ola_task-extra`.